mula/scheduler/schedulers/queue/pq.py

```diff
@@ -98,7 +98,7 @@ def __init__(
         self.lock: threading.RLock = threading.RLock()
 
     @with_lock
-    def pop(self, filters: storage.filters.FilterRequest | None = None) -> tuple[list[models.Task], int]:
+    def pop(self, filters: storage.filters.FilterRequest | None = None) -> list[models.Task]:
         """Remove and return the highest priority item from the queue.
```

Comment:
This docstring seems to indicate we fetch just one item, the actual call seems to limit at 100 items? Which is to expected, and which is useful?

Reply:
I've updated the defaults to limit it to 1